frontend/ui/network/manager.lua

```diff
@@ -600,44 +609,29 @@ function NetworkMgr:showNetworkMenu(complete_callback)
                 return
             end
         end
-        -- NOTE: Also supports a disconnect_callback, should we use it for something?
-        --       Tearing down Wi-Fi completely when tapping "disconnect" would feel a bit harsh, though...
```

Comment:
I'd preserve that comment in the remaining invocation ;).

Reply:
Re-added